src/user/settings.js

```diff
@@ -96,32 +95,34 @@ module.exports = function (User) {
 		return defaultValue;
 	}
 
+	// Co-pilot assisted code
+	function validatePaginationValue(value, max) {
+		console.log('pagination is saved');
+		if (!value || parseInt(value, 10) <= 1 || parseInt(value, 10) > max) {
+			throw new Error(`[[error:invalid-pagination-value, 2, ${max}]]`);
+		}
+	}
+
+	// Co-pilot assisted code
+	function validateLanguage(value, languageCodes) {
+		console.log('language is saved');
+		if (value && !languageCodes.includes(value)) {
+			throw new Error('[[error:invalid-language]]');
+		}
+	}
+
+	// Co-pilot assisted code
```

Comment:
function looks good and easy to read. the comments are also helpful to better understand the code.